src-java/testing/test-library/src/main/java/org/openkilda/testing/model/topology/TopologyDefinition.java

```diff
@@ -226,6 +227,17 @@ public List<Isl> getRelatedIsls(Switch sw) {
         return isls;
     }
 
+    /**
+     * Get random ISLs between two switches.
```

Comment:
why is random?

Reply:
Because we don't really care which ISL to choose (99% of cases there's only one ISL between switches in our labs). To choose some particular ISL, we need to add some comparator or boolean flag to the method parameters.